SwiftyBluetooth/Source/CentralProxy.swift

```diff
@@ -144,7 +144,7 @@ extension CentralProxy {
         let weakRequest = timer.userInfo as! Weak<PeripheralScanRequest>
         
         if weakRequest.value != nil {
-            self.stopScan()
```

Comment:
Happy to provided some sample code of the use case I'm describing if that makes it more clear :smile